src/main/java/com/atharvadholakia/password_manager/logging/MaskingPatternLayout.java

```diff
@@ -17,7 +17,7 @@ public String doLayout(ILoggingEvent event) {
 
   private String maskSensitiveData(String message) {
 
-    String regex = "(?i)(password\\s*[:=]\\s*)(\\S+)";
+    String regex = "(?i)(password\\s*[:=]\\s*)(?!\\$2[aby]|\\{SHA\\}|[a-fA-F0-9]{64})(\\S+)";
```

Comment:
it would be better to extend this regex to also mask the salt in logs.

Reply:
This regex is to unmask the hashed password. Do we need to mask both the hash and the salt?